### ISSUE TITLE
"acpipowerbutton" and "savestate"

### ISSUE THREAD

**Issue by Zegordo** [toptensoftware/VBoxHeadlessTray#26]
Hello!
I would like to ask: do you have any plans to make a choice - "savestate" or "acpipowerbutton" when the computer is shut down?
Because do not always need to save the state, sometimes need a clear reboot of vbox machines

**Comment by simongymer** [comment]
Yes, that would be great as I nearly always a clean shutdown rather than save state because one of my VMs doesn't work with the savestate and i have to reset it on boot up.
